### PR TITLE
Enable/disable express payment methods in Adyen Settings

### PR DESCRIPTION
**Pull request by amihajlovski**
## Summary
Added separate express checkout component in the Business Manager for enabling/disabling express payment methods and ordering

### CODE REVIEW

File: metadata/site_import/meta/system-objecttype-extensions.xml [hunk 1]
```diff
@@ -469,6 +465,34 @@
                 <min-length>0</min-length>
                 <field-length>0</field-length>
             </attribute-definition>
+            <attribute-definition attribute-id="ExpressPayments_enabled">
+                <display-name xml:lang="x-default">Enable express payments</display-name>
```
Comment (zaiddreakh) [inline metadata/site_import/meta/system-objecttype-extensions.xml:469]: Let's make the display names for the new attributes here more descriptive (full sentences) as they will be visible "as is" in the classic config page.

Comment (amihajlovski) [reply metadata/site_import/meta/system-objecttype-extensions.xml:469]: I changed those to more descriptive sentences, please check.

**Review by zaiddreakh — APPROVED** [review]

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+async function handleExpressPaymentsVisibility() {
+  const { expressMethodsOrder } = window;
+  if (expressMethodsOrder) {
+    const sortOrder = expressMethodsOrder.split(',');
+    const container = document.getElementById('express-container');
+    let toSort = container.children;
+    toSort = Array.prototype.slice.call(toSort, 0);
```
Comment (shanikantsingh) [inline src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js:7]: could we use "container.children" as "call" function parameter then we would not need line 6 assignment.

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    toSort.sort(
+      (a, b) =>
+        sortOrder.indexOf(a.dataset.method) -
+        sortOrder.indexOf(b.dataset.method),
+    );
+    container.innerHTML = '';
+    for (let i = 0; i < toSort.length; i += 1) {
```
Comment (shanikantsingh) [inline src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js:14]: we could use "toSort.map" in place of for loop here.

**Review by shanikantsingh — CHANGES_REQUESTED** [review]

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js (hunk 2 above)
Comment (amihajlovski) [reply src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js]: great suggestion 👍 done!

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js (hunk 3 above)
Comment (amihajlovski) [reply src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js]: using spread operator here 👍

**Review by zaiddreakh — APPROVED** [review]

**Review by shanikantsingh — APPROVED** [review]

**Review by shanikantsingh — APPROVED** [review]

**Pull request merged**